src/attesters/sev-snp/sev_snp.h

```diff
@@ -0,0 +1,78 @@
+/* Copyright (c) 2022 Intel Corporation
+ * Copyright (c) 2020-2022 Alibaba Cloud
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef _SEV_SNP_H
+#define _SEV_SNP_H
+
+#include <stdint.h>
+#include <stdio.h>
+#include <rats-tls/api.h>
+
```

Comment:
You don't need to put "Refer to https://www.amd.com/system/files/TechDocs/56860.pdf" everywhere. Just Leave a single line such as "/* The following structures are defined by xxxx, please refer to https://www.amd.com/system/files/TechDocs/56860.pdf for details."